# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../pan/gcc43.patch
# Copyright (C) 2010 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

Description: Fixes compiling using gcc >= 4.3

--- pan-0.133/pan/general/log.cc.orig	2010-08-13 17:34:17.228176099 +0200
+++ pan-0.133/pan/general/log.cc	2010-08-13 17:34:29.340007660 +0200
@@ -20,6 +20,7 @@
 #include <config.h>
 #include <iostream>
 #include <cstdarg>
+#include <cstdio>
 #include "log.h"
 
 using namespace pan;
--- pan-0.133/pan/general/progress.cc.orig	2010-08-13 17:36:11.700003004 +0200
+++ pan-0.133/pan/general/progress.cc	2010-08-13 17:36:22.568023413 +0200
@@ -19,6 +19,7 @@
 
 #include <config.h>
 #include <cstdarg>
+#include <cstdio>
 #include "progress.h"
 #include "string-view.h"
 
--- pan-0.133/pan/usenet-utils/numbers.cc.orig	2010-08-13 18:10:31.856010852 +0200
+++ pan-0.133/pan/usenet-utils/numbers.cc	2010-08-13 18:11:03.836003223 +0200
@@ -19,6 +19,7 @@
 
 #include <config.h>
 #include <cctype>
+#include <cstdio>
 #include <pan/general/string-view.h>
 #include "numbers.h"
 #include <algorithm>
--- pan-0.133/pan/tasks/nntp.cc.orig	2010-08-13 18:36:41.276089959 +0200
+++ pan-0.133/pan/tasks/nntp.cc	2010-08-13 18:38:00.248090443 +0200
@@ -21,6 +21,7 @@
 #include <cassert>
 #include <cstdarg>
 #include <cstdlib> // abort, atoi, strtoul
+#include <cstdio>  // snprintf
 extern "C" {
   #include <glib.h>
   #include <glib/gi18n.h>
--- pan-0.133/pan/tasks/nntp-pool.cc.orig	2010-08-13 18:40:17.412253225 +0200
+++ pan-0.133/pan/tasks/nntp-pool.cc	2010-08-13 18:40:47.688337077 +0200
@@ -19,6 +19,7 @@
 
 #include <config.h>
 #include <ctime>
+#include <cstdio>
 #include <glib/gi18n.h>
 #include <pan/general/debug.h>
 #include <pan/general/log.h>
